gsutil.py

```diff
@@ -75,25 +75,25 @@ def OutputAndExit(message):
 # the directory under third_party and the second element is the subdirectory
 # that needs to be added to sys.path.
 THIRD_PARTY_LIBS = [
-    ('argcomplete', ''),  # For tab-completion (gcloud installs only).
-    ('mock', ''),  # mock and dependencies must be before boto.
-    ('funcsigs', ''),  # mock dependency
-    ('google-reauth-python', ''),  # Package name: google_reauth
-    ('pyu2f', ''),  # google_reauth dependency
-    ('oauth2client', ''),  # oauth2client and dependencies must be before boto.
-    ('pyasn1', ''),  # oauth2client dependency
-    ('pyasn1-modules', ''),  # oauth2client dependency
-    ('rsa', ''),  # oauth2client dependency
-    ('apitools', ''),
-    ('boto', ''),
-    ('gcs-oauth2-boto-plugin', ''),
-    ('fasteners', ''), # oauth2client and apitools dependency
-    ('monotonic', ''), # fasteners dependency
-    ('httplib2', submodule_pyvers),
-    ('python-gflags', ''),
-    ('retry-decorator', ''),
-    ('six', ''),
-    ('socksipy-branch', ''),
+  ('argcomplete', ''),  # For tab-completion (gcloud installs only).
```

Comment:
This new indentation is wrong; we use 2-space indent for most cases, but we use 4 when continuing an expression (args to a function, adding items to a collection, etc.), we use 4 spaces.  This is especially helpful for differentiating multiline if statements from their bodies, e.g.:

```python
def Func():
  foo = 'bar'
  my_list = ['default_arg']
  # This is indented by 2 spaces
  if SomeConditionFunction(
      arg1,  # This was indented by 4 more spaces, starting from where the "if" was.
      arg2):
    foo = 'qux'  # Only indented 2 farther than the above "if".
    my_list = [
        # Continuation of the expression on the line above; this is indented by 4.
        'item1',
        'item2',
        'etc'
    ]
```